techniques/index.html

```diff
@@ -48,6 +48,7 @@ <h1 id="title" class="title p-name">Techniques for WCAG 2.2</h1>
 			<h2>Change Log</h2>
 			<p>A list of new, removed or significantly updated techniques:</p>
 			<ol>
+				<li><time datetime="2024-05-04">4 May 2024</time>: Removed F87 Failure of Success Criterion 1.3.1 due to inserting non-decorative content by using :before and :after pseudo-elements and the 'content' property in CSS</li>
```

Comment:
per discussion last week, should we just remove the changelog all together since it's inconsistently appearing in techniques and is not always updated?